Lib/test/test_crypt.py

```diff
@@ -39,12 +39,26 @@ def test_methods(self):
         else:
             self.assertEqual(crypt.methods[-1], crypt.METHOD_CRYPT)
 
+    @unittest.skipUnless(crypt.METHOD_SHA256 in crypt.methods or
+                         crypt.METHOD_SHA512 in crypt.methods,
+                        'requires support of SHA-2')
+    def test_sha2_rounds(self):
+        for method in (crypt.METHOD_SHA256, crypt.METHOD_SHA512):
+            for rounds in 1000, 10000, 100000:
```

Comment:
Would you mind to write: "1_000, 10_000, 100_000"? It would be easier to read :-)